tests/realtikvtest/addindextest/operator_test.go

```diff
@@ -217,6 +224,7 @@ func TestBackfillOperatorPipeline(t *testing.T) {
 	require.NoError(t, err)
 	require.Len(t, keys, 10)
 	require.Len(t, values, 10)
+	require.Equal(t, int64(10), totalRowCount.Load())
```

Comment:
Does this line mean we can remove L225-L226?

Reply:
Removed.